aspnetcore/mvc/controllers/filters.md

```diff
@@ -17,6 +17,8 @@ By [Tom Dykstra](https://github.com/tdykstra/) and [Steve Smith](https://ardalis
 
 *Filters* in ASP.NET Core MVC allow you to run code before or after certain stages in the request processing pipeline.
 
+This topic applies to ASP.NET Core MVC and not Razor Pages. ASP.NET Core 2.1 preview and later supports `IPageFilter` and `IAsyncPageFilter` for Razor Pages.
```

Comment:
I suggest putting this text inside of an IMPORTANT widget to make it stand out. The text blends in right now and could easily be missed by someone skimming the doc.